fastlane/playstoreres/metadata/fr-FR/play_store_screenshot_7.txt

```diff
@@ -0,0 +1 @@
+Téléverser en déplacement
```

Comment:
I think nobody in France says "Téléverser" in practice tbh. It might be the official work the French Academy registers as the official translation for "upload", but very few people use it in practice and I'm not even sure everybody speaking French know that word and know what it means 🤔

French people usually know the anglicism "uploader" (i.e. english verb "to upload" made as a French verb ending in "-er") though not sure what's best (using the official word that some French speakers might not know the meaning of, or using the anglicism that everybody uses even if not academical)

Anyway, could be worth asking some French speakers in Editorial

Reply:
EDIT: Actually nevermind, I circled back with some Frenchmatticians and it seems that that debate has already been had (see [here](https://fr.wordpress.org/2017/11/03/traduction-du-terme-upload-pour-wordpress-en-francais/)) and considering all that they ended up deciding to still use "Téléverser" throughout WordPress fr_FR, so that makes sense it's used there then (even if it still feels strange to me… but at least that means we're consistent in our vocabulary used for it throughout the app)